cypress/e2e/pages/mediaAssetPage/index.cy.js

```diff
@@ -1,11 +1,353 @@
+    service: 'hindi',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/hindi/sport/2016/08/160822_tc2_testmap1?renderer_env=test', // TC2 video
+    service: 'hindi',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/russian/av/media-45527896', // CPS video with redirect
+    service: 'russian',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/russian/av/media-23320267', // CPS video with redirect
+    service: 'russian',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/russian/news/2016/05/160510_tc2_testmap3?renderer_env=test', // TC2 video
+    service: 'russian',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/serbian/cyr/srbija-52895074', // CPS video
+    service: 'serbian',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/serbian/cyr/23279016', // CPS video
+    service: 'serbian',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/serbian/lat/srbija-52895074', // CPS video
+    service: 'serbian',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/serbian/lat/23279016', // CPS video
+    service: 'serbian',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/swahili/gnb-51703829', // CPS MAP with video clip
+    service: 'swahili',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/swahili/michezo/2016/07/160713_tc2_testmap2?renderer_env=test', // TC2 MAP with audio clip
+    service: 'swahili',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/tigrinya/news-23263262', // CPS MAP with video clip
+    service: 'tigrinya',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/urdu/entertainment-51584098', // CPS MAP with audio clip
+    service: 'urdu',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/simp/world-53107744', // CPS video
+    service: 'zhongwen',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/simp/uk-23283128', // CPS Audio
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/simp/multimedia/2016/11/161107_tc2_testmap1?renderer_env=test', // TC2 Video
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/trad/world-53107744', // CPS video
+    service: 'zhongwen',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/trad/uk-23283128', // CPS Audio
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/trad/multimedia/2016/11/161107_tc2_testmap1?renderer_env=test', // TC2 Video
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+];
+
+// non Amp Non Smoke Test Suites
```

Comment:
What are these tests? Perhaps we can change how this was meant to work? If they were non-smoke tests, perhaps we can include them in the canonical non-smoke tests, and also run AMP tests on them?

Reply:
OK let me have a look

Reply:
It looks like they are TC2 MAPs which appear not to support AMP. Perhaps we can rename this test suite to make it clear that's the case?